### DISCUSSION TITLE
Json from Presets

### DISCUSSION THREAD

**Discussion by droidusmcmoidus** [matteobruni/tsparticles#2454] [Q&A] [answered]
Hey - Im using elementor and PA to import Jsons from the demos... is it possible to add the sea anemone preset as a demo or to generate a json from that preset somehow?

Thanks.

**Comment by matteobruni** [comment, marked as answer]
Hi @droidusmcmoidus,

I don't know which version is used in the elemento plugins. The Sea Anemone preset uses some additional packages like the curves path plugin, and some of the features in the core library were not present in some early versions.

You can try asking them more details about it, and how to add additional scripts.

**Comment by droidusmcmoidus** [comment]
OK Matteo, thanks for your help.
